mediator/med_phases_restart_mod.F90

```diff
@@ -300,10 +304,21 @@ subroutine med_phases_restart_write(gcomp, rc)
                        nx=nx, ny=ny, nt=1, whead=whead, wdata=wdata, pre=trim(compname(n))//'ExpAccum', rc=rc)
                    if (ChkErr(rc,__LINE__,u_FILE_u)) return
                 endif
+
+                ! Write import field bundle accumulators
+                if (ESMF_FieldBundleIsCreated(is_local%wrap%FBImpAccum(n,n),rc=rc)) then
+                   ! TODO: only write this out if actually have done accumulation
```

Comment:
Should this comment be removed?

Reply:
yes - this is still needed. The accumulation field bundle is created by the present and active flags - but actual accumulation is triggered by run time calls.